### PR TITLE
Add missing links to README.md files of demos and models

### PR DESCRIPTION
**Pull request by nignatovsky**
Changed models/public/index.md and demos/README.md

### CODE REVIEW

**Review by Wovchena — APPROVED** [review]

File: models/public/index.md [hunk 1]
```diff
@@ -154,6 +154,12 @@ SSD-based and provide reasonable accuracy/performance trade-offs.
 | YOLO v4                              | Keras\*                  | [yolo-v4-tf](./yolo-v4-tf/README.md) | 71.17%/75.02% | 128.608	 |	64.33 |
 | YOLO v4 Tiny                         | Keras\*                  | [yolo-v4-tiny-tf](./yolo-v4-tiny-tf/README.md) | | 6.9289 | 6.0535 |
 
+## Face Detection
```
Comment (IRDonch) [inline models/public/index.md:157]: We don't have a separate face detection task type; this section should be merged into Object Detection.

Note that you can find a model's task type in its `model.yml` or `composite-model.yml` file.

File: models/public/index.md [hunk 2]
```diff
@@ -165,6 +171,12 @@ SSD-based and provide reasonable accuracy/performance trade-offs.
 | MobileFaceNet,ArcFace@ms1m-refine-v1 | MXNet\*        | [face-recognition-mobilefacenet-arcface](./face-recognition-mobilefacenet-arcface/README.md) | 98.8695% | 0.449 | 0.993 |
 | SphereFace                           | Caffe\*        | [Sphereface](./Sphereface/README.md) | 98.8321% | 3.504 | 22.671 |
 
+## Gesture Recognition
```
Comment (IRDonch) [inline models/public/index.md:174]: This should be merged into Action Recognition.

File: models/public/index.md [hunk 3]
```diff
@@ -253,6 +265,19 @@ The task of image translation is to generate the output based on exemplar.
 | -----------| -------------- | ---------------------------------- | -------- | --------- | -------- |
 | CoCosNet   | PyTorch\*      | [cocosnet](./cocosnet/README.md) | 12.93dB  | 1080.7032 | 167.9141 |
 
+
+## License Plates Detection
```
Comment (IRDonch) [inline models/public/index.md:269]: This should be merged into Object Detection.

File: models/public/index.md [hunk 4]
```diff
@@ -253,6 +265,19 @@ The task of image translation is to generate the output based on exemplar.
...
+
+| Model Name | Implementation | OMZ Model Name                     | Accuracy | GFlops    | mParams  |
+| -----------| -------------- | ---------------------------------- | -------- | --------- | -------- |
+| vehicle-license-plate-detection-barrier-0123 | TensorFlow\* | [vehicle-license-plate-detection-barrier-0123](./vehicle-license-plate-detection-barrier-0123/README.md) | 	99.52% | 0.271 | 0.547 |
+
+## License Plates Recognition
```
Comment (IRDonch) [inline models/public/index.md:275]: This should be changed to Optical Character Recognition.

File: models/public/index.md [hunk 5]
```diff
@@ -289,6 +314,13 @@ The reported metric is collected over the alphanumeric subset of icdar 13 (1015
 | -------------- | -------------- | ------------------------------------------------------ | -------- | ------- | -------- |
 | Resnet-FC      | PyTorch\*      | [text-recognition-resnet-fc](./text-recognition-resnet-fc/README.md) | 90.94% | 40.3704 | 177.9668  |
 
+## Text to Speech
+
+| Model Name     | Implementation | OMZ Model Name                                         | Accuracy | GFlops  | mParams  |
+| -------------- | -------------- | ------------------------------------------------------ | -------- | ------- | -------- |
+| ForwardTacotron | PyTorch\* | [forward-tacotron](./forward-tacotron/README.md) |  | 6.66 | 13.81 |
+| WaveRNN        | PyTorch\*      | [wavernn](./wavernn/README.md) |  | 0.37 | 0.4 |
```
Comment (IRDonch) [inline models/public/index.md:322]: For composite models, the GFlops/mParams values should be listed on a per-component basic. You can use `mtcnn` as an example.

**Review by IRDonch — CHANGES_REQUESTED** [review]

**Comment by nignatovsky** [comment]
All suggestions were applied.

**Review by IRDonch — APPROVED** [review]

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**